internal/jennies/java/jennies.go

```diff
@@ -39,6 +40,8 @@ func (config *Config) InterpolateParameters(interpolator func(input string) stri
 func (config Config) MergeWithGlobal(global languages.Config) Config {
 	newConfig := config
 	newConfig.generateBuilders = global.Builders
+	// newConfig.generateConverters = global.Converters
+	newConfig.generateConverters = false
```

Comment:
Just checking: is that left on purpose?

Reply:
Yes, because the converter isn't managing yet unknown dataqueries and panels and "generics" needs more love.